seed/views/v3/compliance_metrics.py

```diff
@@ -239,7 +239,8 @@ def evaluate(self, request, pk):
                 'message': 'ComplianceMetric does not exist'
             }, status=status.HTTP_404_NOT_FOUND)
 
-        response = compliance_metric.evaluate()
+        user_ali = AccessLevelInstance.objects.get(pk=request.access_level_instance_id)
```

Comment:
is this guaranteed to always return an `ALI`?

Reply:
yes after this whole AH feature is implemented